_data/events.yml

```diff
@@ -1,3 +1,10 @@
+- title: Luxury Automation For Enterprise Cloud Computing
+  location: "YOOX NET-A-PORTER GROUP - TECH HEADQUARTER, Via Nannetti 1, 40069 Zola Predosa (BO)"
+  date: 2016-06-07 18:00:00
+  host: YOOX NET-A-PORTER GROUP
+  link: https://www.eventbrite.it/e/biglietti-luxury-automation-for-enterprise-cloud-computing-34621565033
+  description: "Onces again YOOX NET-A-PORTER GROUP will open the doors of its Technology departments to share with you its experience around cloud, automation and containers.The purpose of this event is to give you an overview of how we have automated our infrastructure builds. At the end of this session, you will be able to take this knowledge to use in your own projects. As well as YNAP presenters, we have the pleasure of two guest speakers: Luca Villa, Senior Solution Architect, who will give a presentation about the Red Hat approach to the open source automation and Giovanni Bacca, Hybrid IT Technology Consultant, who will explain how Hewlett Packard Enterprise manages the “cohabitation" problem between DB and Docker."
+  gmaps: <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2845.9486972797813!2d11.23666871602944!3d44.49571800555854!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x477fd6ef44a6a65b%3A0x5ce82e081c8a9a53!2sYOOX+NET-A-PORTER+GROUP!5e0!3m2!1sit!2sit!4v1496051954722" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
```

Comment:
potresti spostarlo in fondo al file in modo che gli eventi siano ordinati per date?
grazie.